init_query.sql

```diff
@@ -43,8 +43,13 @@ LEFT JOIN
     utterances AS utt  -- utterance-level stats such as speech rate.
 ON
     phone.u_ID = utt.u_id
+LEFT JOIN
+	(
+	    SELECT stdev(u.log_speech_rate) AS speech_rate, AVG(u.log_speech_rate) AS avg_speech_rate FROM utterances as u
+    ) AS z
 WHERE
-    ws.cldf_languageReference = word.cldf_languageReference AND
+    (ws.cldf_languageReference = word.cldf_languageReference AND ws.cldf_name = word.cldf_name) AND
+    ls.cldf_languageReference = word.cldf_languageReference AND
```

Comment:
The `WHERE` clause condition for the `langstats` view was missing. I suppose that was the issue blowing up the query on your laptop.

Reply:
Ah, I didn't realize I need the where clause. I need to read up a little bit on that

Reply:
There's a bit of a philosophical discussion about whether to join explicitly or implicitly (see https://dataschool.com/how-to-teach-people-sql/difference-between-where-and-on-in-sql/). I got a bit accustomed to use "implicit joins", i.e. `SELECT FROM a, b WHERE a.x = b.y`, but it seems - and you forgetting the WHERE clause might be evidence for this - that explicit joins are more transparent. Normally, I'm a fan of the "principle of locality", so I might switch to explicit joins, where the `ON` clause follows right away in the future.